pallets/identity/src/lib.rs

```diff
@@ -828,8 +840,22 @@ decl_module! {
         }
 
         #[weight = <T as Trait>::WeightInfo::add_investor_uniqueness_claim_v2()]
-        pub fn add_investor_uniqueness_claim_v2(origin, target: IdentityId, claim: Claim, proof: ScopeClaimProof, expiry: Option<T::Moment>) -> DispatchResult {
-            Self::base_add_investor_uniqueness_claim(origin, target, claim, proof.into(), expiry)
+        pub fn add_investor_uniqueness_claim_v2(origin, target: IdentityId, scope: Scope, claim: Claim, proof: ScopeClaimProof, expiry: Option<T::Moment>) -> DispatchResult {
+            Self::base_add_investor_uniqueness_claim(origin, target, claim, Some(scope), proof.into(), expiry)
+        }
+
+        /// Revokes a specific claim using its claim unique index, instead of its
+        /// content.
+        ///
+        /// Please note that only the `issuer` of the target claim can effectively call this function, because its `DID` is part of claim index.
```

Comment:
There is no `issuer` among the arguments. To find it you would need to go to the source code.
`DID` is not an identifier so it shouldn't be monospaced.